charts/datree-admission-webhook/templates/deployment.yaml

```diff
@@ -27,6 +27,12 @@ spec:
       {{- if .Values.nodeSelector }}
       nodeSelector: {{- toYaml .Values.nodeSelector | nindent 8 }}
       {{- end }}
+      {{- if or .Values.affinity }}
```

Comment:
I think you don't have to use the `if` here. you can use only the `with` here. it is good enough IMO though